src/app/index.js

```diff
@@ -59,6 +46,70 @@ export default yeoman.generators.Base.extend({
   },
 
   /**
+   * Gets the name of the scope including the "@" symbol.
+   *
+   * @param  {String} scope
+   * @return {String}
+   */
+  _getScope(scope) {
+    return scope && typeof scope === 'string' ? `@${scope}` : '';
```

Comment:
should this make sure you don't get double `@` or is there extra checking for that elsewhere?